stdlib/fileinput.pyi

```diff
@@ -19,40 +23,133 @@ __all__ = [
     "hook_encoded",
 ]
 
-if sys.version_info >= (3, 9):
-    from types import GenericAlias
+if sys.version_info >= (3, 11):
+    _TextMode: TypeAlias = Literal["r"]
+else:
+    _TextMode: TypeAlias = Literal["r", "rU", "U"]
+
+_AnyStr_co = TypeVar("_AnyStr_co", str, bytes, covariant=True)
```

Comment:
We define the same `TypeVar` (with the same name) in `builtins.pyi` and `os/__init__.pyi` — is it worth adding it to `_typeshed`, do you think?

Reply:
It's simple enough that I don't mind redefining it, but I don't feel strongly.

Reply:
We can leave it for now